src/github.com/stellar/go-horizon/resource_account.go

```diff
@@ -82,3 +129,10 @@ func AmountToString(amount int64) string {
 	frac := amount % stellarbase.One
 	return fmt.Sprintf("%d.%07d", whole, frac)
 }
+
+func IsTrue(value int32) bool {
```

Comment:
Or, we can take it even further (just thinking out loud here):
```go
type Flags int

func (f Flags) Has(mask int) bool {
  return (f & mask) != 0
}
```

Which provides a nice interface, IMO:

```go
accountFlags := Flags(ac.Flags)
accountFlags.Has(db.FlagAuthRequired)
accountFlags.Has(db.FlagAuthRevocable)
```